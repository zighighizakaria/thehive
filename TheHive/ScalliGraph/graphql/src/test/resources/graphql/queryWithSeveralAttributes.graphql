query AllPerson {
  allPeople {
    sort(
      sort:{
        name: incr
      }) {
      toList {
        name
        age
      }
    }
  }
}