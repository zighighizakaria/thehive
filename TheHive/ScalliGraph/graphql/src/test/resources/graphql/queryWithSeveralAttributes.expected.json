{
  "data": {
    "allPeople": {
      "sort": {
        "toList": [
          {"name": "franck", "age": 28},
          {"name": "josh", "age": 32},
          {"name": "marc", "age": 34},
          {"name": "marko", "age": 29},
          {"name": "peter", "age":35},
          {"name": "vadas", "age": 27}
        ]
      }
    }
  }
}